misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -13,6 +13,7 @@ data class WebConfig(
   val selectors: Int? = null,
   val acceptors: Int? = null,
   val queueSize: Int? = null,
+  val jettyMaxThreadPoolSize: Int? = null,
```

Comment:
`jetty_max_thread_pool_size`

Reply:
hmm ideally we migrate queueSize to snake_case as well, assuming it'd be an easy migration because I imagine contacts is the only client

Reply:
I'll change both